README.md

```diff
@@ -30,6 +30,17 @@ You can find all dependencies in requirements.txt
 * [Web Search](https://pypi.org/project/microsoft-bing-websearch/).
 * required packages: "msrest>=0.5.0", "msrestazure>=0.4.32,<2.0.0", "azure-common~=1.1"
 
+### Running tests
+First execute the following command from the root level of the repo:
+```bash
+pip install -r ./tests/test-requirements.txt
+```
+
+Then run pytest by simply typing it into your CLI:
+```bash
+pytest
+```
+
```

Comment:
This doesn't seem right, review the command locally then accommodate the changes.

Reply:
added a missing init file in the tests directory for this to work

Reply:
This still won't work, try creating a new virtual environment, and run the commands you mentioned. See the issues, fix them, and come back. Good luck.

Also, I don't think the additional `__init__` file was necessary.

Reply:
This works perfectly, why are you suggesting it won't work have you tried it?

Also the init file was necessary otherwise the tests won't run!

This was only meant to document how to run the tests locally not to actually get the tests to pass.

Have you actually tried this out or are you only commenting based on what you see?